modules/system-management/pages/memory-management.adoc

```diff
@@ -22,32 +22,76 @@ See xref:gsql-ref:querying:distributed-query-mode.adoc[].
 You can monitor memory usage by query and by machine.
 
 === Monitor query memory usage
-TigerGraph's Graph Processing Engine (GPE) records memory usage by query at different stages of the query and saves it to `$(gadmin config get System.LogRoot)/gpe/log.INFO`.
+TigerGraph's Graph Processing Engine (GPE) records memory usage by query at different stages of the query and saves it to the GPE logs at `$(gadmin config get System.LogRoot)/gpe/log.INFO`.
+This is an estimation of the query's memory usage *on a particular node* in a TigerGraph cluster.
+
 You can monitor how much memory a query is using by searching the log file for the request ID and filter for lines that contain `"QueryMem"`:
 
-[source,console]
+[source.wrap,console]
 ----
-$ grep -i <request_id> $(gadmin config get System.LogRoot)/gpe/log.INFO | grep -i "querymem"
+$ grun all 'grep -i <request_id> $(gadmin config get System.LogRoot)/gpe/log.INFO | grep -i "querymem"'
 ----
 
 You can also run a query first, and then run the following command immediately after to retrieve the most recent query logs and filter for `"QueryMem"`:
 
 [source,console]
 ----
-$ tail -n 50 $(gadmin config get System.LogRoot)/gpe/log.INFO |
-    grep -i "querymem"
+$ grep 'tail -n 50 $(gadmin config get System.LogRoot)/gpe/log.INFO |
+    grep -i "querymem"'
 ----
 
+==== Interpreting log response
+
 The logs show memory usage by the query at different stages of its execution.
-The number at the end of each line indicates the number of bytes of memory utilized by the query:
+The number at the end of each line indicates the number of bytes of memory utilized by the query.
 
-[source.wrap,console]
+[WARNING]
```

Comment:
Maybe use `limitation` instead of warnings?

Reply:
Ahh, WARNING is a keyword for antora. Maybe I can change it to CAUTION